intro-basics-2023-1/Assets/Scripts/EnemyTankShooter.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyTankShooter : MonoBehaviour
+{
+    [SerializeField]
+    private GameObject _projectilePrefab;
+
+    [SerializeField]
+    private Transform[] _shootPoints;
+
+    private float timeRemaining = 3;
+    
+
+    void Update()
+    {
+        if (timeRemaining > 0)
+        {
+            timeRemaining -= Time.deltaTime;
+        }
+        else
+        {
+            Debug.Log("Time has run out");
+            timeRemaining = 3;
+            GameObject projectile1 = Instantiate(_projectilePrefab);
+            projectile1.transform.position = _shootPoints[0].position;
```

Comment:
La ventaja de utilizar un array `[]` es que con un `for` puedes escribir el código una solo vez y aplicarlos a todos los elementos del array.